atd-vze/src/views/Crashes/CrashDiagram.js

```diff
@@ -12,13 +12,14 @@ import {
 } from "reactstrap";
 import axios from "axios";
 import { TransformWrapper, TransformComponent } from "react-zoom-pan-pinch";
+import { isDev } from "../../helpers/environment";
 
 const CrashDiagram = props => {
   const [rotation, setRotation] = useState(0);
 
   // Set S3 folder for diagram depending on environment
   const s3Folder =
-    process.env.NODE_ENV === "production" ? "production" : "staging";
+    process.env.NODE_ENV === "production" || isDev ? "production" : "staging";
```

Comment:
This line updates the app to target the production S3 path for CR3 diagrams when using a production snapshot in the local stack.